rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/EntityResource.java

```diff
@@ -221,6 +221,11 @@ public int compare(Task<?> o1, Task<?> o2) {
     @Override
     public TaskSummary getTask(final String application, final String entityToken, String taskId) {
         // TODO deprecate in favour of ActivityApi.get ?
+        Entity entity =brooklyn().getApplication(application);
+        if (entity != null && !Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_ENTITY, entity)) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to see the entity '%s'",
```

Comment:
Message should say `User '%s' is not authorized to see the task '%s' for the entity '%s'`
Please also check the parameters for the templated message